Dockerfile

```
@@ -33,7 +33,7 @@ LABEL summary="$SUMMARY" \
 # gcc to compile some python packages (e.g. ciso8601)
 # shadow-utils to make useradd available
 # libpq-devel needed for building psycopg2
-RUN INSTALL_PKGS="python39 python39-devel glibc-langpack-en gcc shadow-utils libpq-devel" && \
+RUN INSTALL_PKGS="python3.11 python3.11-devel glibc-langpack-en gcc shadow-utils libpq-devel" && \
```

Looks like we need the C++ compiler now in order successfully build the wheel for pandas.
```suggestion
RUN INSTALL_PKGS="python3.11 python3.11-devel glibc-langpack-en gcc gcc-c++ shadow-utils libpq-devel" && \
```